lib/main.dart

```diff
@@ -1,67 +1,315 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-
+import 'package:flutter/services.dart';
 void main() {
-  runApp(const MyApp());
+  WidgetsFlutterBinding.ensureInitialized();
+  SystemChrome.setPreferredOrientations([DeviceOrientation.landscapeLeft])
+      .then((_) {
+  runApp(MyApp());
+  });
 }
 
+var marginDivider = 1;
+double orientationWidth = 1;
+double orientationHeight = 1;
+
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
-
-  // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const HomePage(),
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({Key? key, required this.title}) : super(key: key);
-  final String title;
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
 
   @override
-  State<MyHomePage> createState() => _MyHomePageState();
+  _HomePageState createState() => _HomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+Widget _imageContainerStart(BuildContext context, String link, String gameName,
+    String gameShortDescription) {
+  var phoneSize = MediaQuery.of(context).size;
+  if (MediaQuery.of(context).orientation == Orientation.landscape) {
+    orientationWidth = 0.9;
+    orientationHeight = 1.24;
+    marginDivider = 2;
+  }else{
+    orientationWidth = 1;
+    orientationHeight = 1;
+    marginDivider = 1;
+  }
+  return Container(
+      width: phoneSize.width * 0.83 * orientationWidth,
```

Comment:
You often check for device height/width. Instead, you should try using flex layouts with constraints (instead of setting fixed width/height, you simply instruct your control to occupy the entire available space, and then you define what space is available using constraints)